### ISSUE TITLE
Create a README documenting provided functionality

### ISSUE THREAD

**Issue by skleinjung** [jgornick/rnw-starter#5]
It would be useful to get an overview on the project's home page describing what frameworks, tools, etc. this starter project includes. A brief description of the purpose of those tools, and some of the common customizations a new project would need to perform would be helpful as well.

**Comment by skleinjung** [comment]
I did just see the SETUP.md, which provides some of the information I mentioned above. But having that document (or a more marketing-friendly version of it) be the first thing people see would be an improvement.

**Comment by jgornick** [comment]
Totally, not there yet to create something more formal. Will get there once the implementation is complete.

**Comment by skleinjung** [comment]
Yup, figured as much. Just one of those things that's easier to do as it evolves, instead of in a "big bang" at the end where you may forget what you added 6 months ago. Also, I was selfishly trying to figure out if I could use what was here as a starter template yet or not. :D
